Documentation/rbd-mirroring.md

```diff
@@ -0,0 +1,247 @@
+ authenticate to each other:
+
+* For Bootstrapping a peer cluster its bootstrap secret is required. To determine the name of the secret that contains the bootstrap secret execute the following command on the remote cluster (site-b)
+
+```bash
+kubectl get cephblockpool.ceph.rook.io/mirroredpool -n rook-ceph --context=cluster-b -ojsonpath='{.status.info.rbdMirrorBootstrapPeerSecretName}'
+```
+
+> ```bash
+> pool-peer-token-mirroredpool
+> ```
+
+Here, `pool-peer-token-mirroredpool` is the desired bootstrap secret name.
+
+* The secret pool-peer-token-mirroredpool contains all the information related to the token and needs to be injected to the peer, to fetch the decoded secret:
+
+```bash
+kubectl get secret -n rook-ceph pool-peer-token-mirroredpool --context=cluster-b -o jsonpath='{.data.token}'|base64 -d
+```
+
+> ```bash
+>eyJmc2lkIjoiNGQ1YmNiNDAtNDY3YS00OWVkLThjMGEtOWVhOGJkNDY2OTE3IiwiY2xpZW50X2lkIjoicmJkLW1pcnJvci1wZWVyIiwia2V5IjoiQVFDZ3hmZGdxN013R0JBQWZzcUtCaGpZVjJUZDRxVzJYQm5kemc9PSIsIm1vbl9ob3N0IjoiW3YyOjE5Mi4xNjguMzkuMzY6MzMwMCx2MToxOTIuMTY4LjM5LjM2OjY3ODldIn0=
+> ```
+
+* Get site name from secondary cluster(cluster-b)
+
+```bash
+kubectl get cephblockpools.ceph.rook.io mirroredpool -nrook-ceph --context=cluster-b -o jsonpath='{.status.mirroringInfo.site_name}'
+```
+
+> ```bash
+> 5a91d009-9e8b-46af-b311-c51aff3a7b49
+> ```
+
+* With this Decoded value, create a secret on the primary site(cluster-a), using the site name of the peer as the name.
+
+```bash
+kubectl -n rook-ceph create secret generic --context=cluster-a 5a91d009-9e8b-46af-b311-c51aff3a7b49 --from-literal=token=eyJmc2lkIjoiNGQ1YmNiNDAtNDY3YS00OWVkLThjMGEtOWVhOGJkNDY2OTE3IiwiY2xpZW50X2lkIjoicmJkLW1pcnJvci1wZWVyIiwia2V5IjoiQVFDZ3hmZGdxN013R0JBQWZzcUtCaGpZVjJUZDRxVzJYQm5kemc9PSIsIm1vbl9ob3N0IjoiW3YyOjE5Mi4xNjguMzkuMzY6MzMwMCx2MToxOTIuMTY4LjM5LjM2OjY3ODldIn0= --from-literal=pool=mirroredpool
+```
+
+> ```bash
+> secret/5a91d009-9e8b-46af-b311-c51aff3a7b49 created
+> ```
+
+* This completes the bootstrap process for cluster-a to be peered with cluster-b
+* Repeat the process switching cluster-b in place of cluster-a, to complete the bootstrap process across both peer clusters.
+
+For more details, refer to the official rbd mirror documentation on
+ [how to create a bootstrap peer](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#bootstrap-peers).
+
+## Configure the RBDMirror Daemon
+
+Replication is handled by the rbd-mirror daemon. The rbd-mirror daemon
+ is responsible for pulling image updates from the remote, peer cluster,
+ and applying them to image within the local cluster.
+
+Creation of the rbd-mirror daemon(s) is done through the custom resource
+ definitions (CRDs), as follows:
+
+* Create mirror.yaml, to deploy the rbd-mirror daemon
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephRBDMirror
+metadata:
+  name: my-rbd-mirror
+  namespace: openshift-storage
+spec:
+  # the number of rbd-mirror daemons to deploy
+  count: 1
+  peers:
```

Comment:
We want users to use `cephblockpool` to add peer secrets.

Reply:
+1 we have recently changed that I don't recommend to use the `CephRBDMirror` resource, please use `CephBlockPool`.

Reply:
Thanks for pointing it out. Updated the doc with the required changes!